### ISSUE TITLE
consider running md5 check on generated cache files

### ISSUE THREAD

**Issue by mpareja** [meet-tarantino/tarantino#8]
It might be trivial:

`md5sum "$services_cache_file" >> "$md5check"`

**Comment by darrenmce** [comment]
is this for if someone manually edits the cache file?

**Comment by mpareja** [comment]
Yeah, kinda of silly really. It seemed trivial to implement but thought it warranted discussion (like your above comment) rather than dropping the change in with an active PR.
